src/integ_test_resources/android/sdk/integration/cdk/README.md

```diff
@@ -20,25 +20,39 @@ npm install -g aws-cdk
 Ensure that you have [credentials in your environment sufficient to run
 the CDK](https://docs.aws.amazon.com/cdk/latest/guide/getting_started.html#getting_started_credentials).
 
-Install CDK dependencies:
+Install CDK dependencies from the root directory of the repo:
 ```console
 # Best to do this in a virtualenv
-pip install -r requirements.txt
+pip3 install -r requirements.txt
 ```
 
-Generate a Pinpoint stack template at
+**Export credentials and region:**
+
+```console
+isengard assume <account name>
+isengard credentials <account name>
+```
+
+Copy and paste the export statements generated. Then paste these export commands:
+
+```console
+export AWS_REGION=us-east-1
+export AWS_DEFAULT_REGION=us-east-1
```

Comment:
Hm. Why do we have two? Can we get back to just one, instead of documenting both?

Reply:
John wasn't sure which was used so just put both for sake of time - can always trim back once we find out which one it is.

Reply:
We'll forget, though, and this says "both are needed." My script used `AWS_DEFAULT_REGION`. What would use `AWS_REGION`? If we can't find a usage now, let's just remove it.

Reply:
Fixed